packages/good-design/src/apps/goodid/wizards/SegmentationWizard.tsx

```diff
@@ -72,10 +79,20 @@ export const SegmentationWizard = (props: SegmentationProps) => {
     }
   };
 
+  const onDispute = async (disputedValues: string[]) => {
+    // should report analytics
+    // todo: replace with analytics report, log for 'unused var' eslint
+    console.log("disputedValues", disputedValues);
+  };
+
   return (
     <WizardContextProvider>
       <Wizard header={<WizardHeader onDone={modalOnDone} error={error} />}>
-        <SegmentationScreenWrapper onDone={props.onDone} onLocationRequest={modalOnLocation} />
+        <SegmentationScreenWrapper onDone={props.onDone} onLocationRequest={modalOnLocation} account={account} />
```

Comment:
I dont see onDone is used
Also what about the SegmentationConfirmation? what happens after that, shouldnt onDone be called?

Reply:
1. because SegmentationScreen uses a localised modal. which I did because the 'location permission denied' modal did not have any relevance for any of the other screens in the flow
2. no, nextStep is called there. the next step is to checkAvailableOffers --> https://github.com/GoodDollar/GoodWeb3-Mono/pull/158